src/feature/coffeeChat/ui/CompanyCoffeeChatListView.tsx

```diff
@@ -0,0 +1,212 @@
+                handleSelectChat(coffeeChat.id);
+              }}
+              disabled={coffeeChat.coffeeChatStatus !== 'WAITING'}
+            />
+            <div
+              key={coffeeChat.id}
+              className="flex flex-1 h-[50px] px-6 justify-between items-center cursor-pointer bg-white rounded-xl duration-300 hover:shadow-md px-4"
+              onClick={() => {
+                toCoffeeChatDetail({ coffeeChatId: coffeeChat.id });
+              }}
+            >
+              <div className="flex items-center gap-2">
+                {coffeeChat.coffeeChatStatus === 'WAITING' && (
+                  <img src={ICON_SRC.BADGES} />
+                )}
+                <span className="text-grey-darker font-regular text-14 truncate">
+                  {coffeeChat.applicant.name}
+                </span>
+                <span className="text-grey-dark-hover font-regular text-12 truncate">
+                  {coffeeChat.title}
+                </span>
+              </div>
+              <div className="flex items-center gap-2">
+                <span className="text-sm text-grey-normal">
+                  {getShortenedDate(coffeeChat.createdAt)}
+                </span>
+                <TagCoffeeChat coffeeChatStatus={coffeeChat.coffeeChatStatus} />
+              </div>
+            </div>
+          </div>
+        ))
+      ) : (
+        <>
+          {Array.from({ length: 12 }).map((_, idx) => (
+            <div
+              key={`loading-${idx}`}
+              className="flex px-[24px] h-[50px] justify-between items-center cursor-pointer bg-white rounded-md"
+            >
+              <Skeleton className="w-[350px] h-[18px]" />
+              <Skeleton className="w-[80px] h-6" />
+            </div>
+          ))}
+        </>
+      )}
+      {responseMessage !== '' && (
+        <FormErrorResponse>{responseMessage}</FormErrorResponse>
+      )}
+      {isModalOpen && (
+        <UpdateCoffeeChatStatusModal
+          onClose={() => {
+            setIsModalOpen(false);
+          }}
+          onConfirm={() => {
+            handleConfirm(modalStatus);
+          }}
+          status={modalStatus}
+          selectedCount={selectedChats.length}
+        />
+      )}
+    </div>
+  );
+};
+
+const useUpdateCoffeeChatStatus = ({
+  setResponseMessage,
+}: {
+  setResponseMessage(input: string): void;
+}) => {
+  const { coffeeChatService } = useGuardContext(ServiceContext);
+  const { token } = useGuardContext(TokenContext);
+  const queryClient = useQueryClient();
+
+  const { mutate: updateCoffeeChatStatus, isPending } = useMutation({
+    mutationFn: ({
+      coffeeChatId,
+      body,
+    }: {
+      coffeeChatId: string;
+      body: CoffeeChatStatusRequest;
+    }) => {
+      if (token === null) {
+        throw new Error('토큰이 존재하지 않습니다.');
+      }
+      return coffeeChatService.updateCoffeeChatStatus({
+        token,
+        coffeeChatId,
+        body,
+      });
+    },
```

Comment:
mutationFn에서는 받기 좋은 형식으로 인자를 받고, 함수 내부에서 body를 만들어도 좋을 것 같습니다.
```suggestion
    mutationFn: ({
      coffeeChatId,
      coffeeChatstatus,
    }: {
      coffeeChatId: string;
      coffeeChatstatus: string; // 알맞은 타입으로 변환
    }) => {
      if (token === null) {
        throw new Error('토큰이 존재하지 않습니다.');
      }
      return coffeeChatService.updateCoffeeChatStatus({
        token,
        coffeeChatId,
        body: {
            coffeeChatStatus
        }
      });
    },
```